### PR TITLE
Initial example for zeto fungible tokens

### PR DESCRIPTION
**Pull request by jimthematrix**

### CODE REVIEW

File: example/zeto/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Example: ZKP Fungible Tokens with Zeto
+
+This example demonstrates a privacy-preserving fungible tokens using [Zeto](https://github.com/hyperledger-labs/zeto) on Paladin.
+
+See the [tutorial](https://lf-decentralized-trust-labs.github.io/paladin/head/tutorials/zeto/) for a detailed explanation.
```
Comment (awrichar) [inline example/zeto/README.md:5]: This tutorial doesn't exist yet, does it?

File: example/zeto/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+build/
+src/abis/*.json
```
Comment (awrichar) [inline example/zeto/.gitignore:3]: Not needed (no ABIs copied in here)

**Review by awrichar — APPROVED** [review]
Looks great; thanks for putting this together.

File: example/zeto/README.md (hunk 1 above)
Comment (jimthematrix) [reply example/zeto/README.md:5]: added

**Pull request merged**